css/grid.css

```diff
@@ -10,7 +10,7 @@ body{
 header{
 	display: grid;
 	grid-template-columns: 1fr;
-	grid-template-rows: auto 1fr auto;
+	grid-template-rows: auto auto auto;
```

Comment:
The measure "1fr" makes sure that it will take all the remaining height when the browser does the calculations.
Measurements set to "auto" will adapt to their content, and measurement "1fr" will use whatever space is left. If possible, restore back to "1fr".